R/primary-keys.R

```diff
@@ -16,12 +16,15 @@
 #' @param force Boolean, if `FALSE` (default), an error will be thrown if there is already a primary key
 #'   set for this table.
 #'   If `TRUE`, a potential old `pk` is deleted before setting a new one.
+#' @param autoincrement
+#'   `r lifecycle::badge("experimental")`
+#'   If `TRUE`, the  column specified in `columns` will be populated
+#'   automatically with a sequence of integers.
```

Comment:
This doesn't work for local dm's yet. MAybe we should mention that in the documentation. Also this should be another TODO in my opinion.